### PR TITLE
Integrate matomo support

### LINKED ISSUE #78: Support matomo environment variables
There are two new environment variables that enable matomo webstats in the RSD:

https://github.com/research-software-directory/RSD-as-a-service/commit/99cff6294a2d0a9c75a6aceeecb59f1c9fda0042

### PR DESCRIPTION
**Pull request by cmeessen**
Fixes #78

### CODE REVIEW

File: templates/rsd-secrets.env.j2 [hunk 1]
```diff
@@ -141,3 +141,8 @@ CROSSREF_CONTACT_EMAIL={{ rsd_crossref_contact_email }}
 # consumed by: frontend
 # ID for Hotjar Tracking Code (needs to be a number)
 HOTJAR_ID={{ rsd_hotjar_id }}
+
+# consumed by: frontend
+# URL (should end with a trailing slash) and ID for Matomo Tracking Code
+MATOMO_URL={{ rsd_matomo_url }}
+MATOMO_ID={{ rsd_matomo_id }}
```
Comment (tobiashuste) [inline templates/rsd-secrets.env.j2:148]: Is it safe to specify the variable empty or does it need to be unset if Matomo is not configured?

**Review by tobiashuste — COMMENTED** [review]
@tharun634 Thanks for implementing. Looks good from my side. :+1: 

I added only one question.

File: templates/rsd-secrets.env.j2 (hunk 1 above)
Comment (cmeessen) [reply templates/rsd-secrets.env.j2:149]: It is ok if it is empty. This is the check in the code:

```typescript
            this.matomoUrl !== undefined && this.matomoUrl.length !== 0 &&
            this.matomoId !== undefined && this.matomoId.length !== 0 &&
```

Comment (tobiashuste) [reply templates/rsd-secrets.env.j2:149]: Thanks for the clarification.

**Review by tobiashuste — APPROVED** [review]
Thanks :airplane: 

I will create a new release soon.

**Comment by cmeessen** [comment]
Great, thank you very much for the quick review :)

**Pull request merged**